ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,295 @@
+import pytest
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from ordered_product.models import OrderedProduct
+from product.models import Product
+from django.core.exceptions import ObjectDoesNotExist
+
+
+@pytest.fixture
+def valid_quantity():
```

Comment:
Why you using a fixture that you can dont need,valid quantities are from 0 and up so you can use it manually